src/kernel.cpp

```diff
@@ -614,42 +614,30 @@ static bool CheckStakeKernelHashV3(CBlockIndex* pindexPrev, unsigned int nBits,
     double coin_age = std::abs((double)nTimeTx-(double)txPrev.nTime);
     double BitsAge = PORDiff * 144;     //For every 100 Diff in Bits, two hours of coin age for researchers
     if (BitsAge > 86400) BitsAge=86400; //Limit to 24 hours (possibility of astronomical diff)
-    // Halford : Explain to the Researcher why they are not staking:
-    if (checking_local && LessVerbose(100))
```

Comment:
Need to stop reviewing that early. Read that code 3 times and realised twice that it was refactored :/